src/atsphinx/typst/writer.py

```
@@ -170,6 +170,84 @@ def _escape(txt: str) -> str:
+                # No need to register for copying - file is already in final location
+                img_node = nodes.image()
+                img_node['uri'] = relative_filename  # Already points to images_dir/filename.svg
+                img_node['alt'] = node.get('alt', 'Mermaid diagram')
+                if 'align' in node:
+                    img_node['align'] = node['align']
+                # Use parent class to render (skip our visit_image path processing)
+                super(TypstTranslator, self).visit_image(img_node)
+                super(TypstTranslator, self).depart_image(img_node)
+        except ImportError:
+            logger.warning("sphinxcontrib.mermaid not installed. Skipping mermaid diagram.")
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)